apps/web/src/pages/templates/workflow/workflow/node-types/WorkflowNode.tsx

```diff
@@ -119,9 +123,10 @@ export function WorkflowNode({
   } = useFormContext();
 
   let stepErrorContent = initialErrors;
-
+  let showGroupError = false;
   if (typeof index === 'number') {
     stepErrorContent = getFormattedStepErrors(index, errors);
+    showGroupError = isStepRoot && hasGroupError(index, errors);
```

Comment:
isSteproot makes sure it is only applied on the workflow node in the editor and not in the variant sidebar